paper.md

```diff
@@ -33,37 +33,41 @@ bibliography: paper.bib
+
+
+# Statement of Need
+
+Neuroimaging is an empirical research area which relies heavily on efficient data acquisition, harmonization, and processing.
+Neuroimaging data sourced from medical imaging equipment, and in particular magnetic resonance imaging (MRI) scanners, can be exported in numerous formats, among which DICOM (Digital Imaging and Communications in Medicine) is most prominent.
+DICOM data are often transmitted to PACS (Picture Archiving and Communication Systems) servers for archiving or further processing.
+Unlike in clinical settings, where data are interfaced with directly from PACS in the DICOM format, in neuroimaging research, tools typically require data files in the much simpler (and metadata-restricted) NIfTI [@nifticlib] format.
+Tools such as `dcm2niix` [@Li_2016] can be used to convert *individual* DICOM files into named NIfTI files, and can extract metadata fields not covered by the NIfTI header into sidecar `.json` files.
+However, the scope of such tools is limited, as it does not extend to organizing *multiple* files within a study.
+
+HeuDiConv was developed to provide flexible tooling so that labs may rapidly and consistently convert collections of DICOM files into collections of NIfTI files in customizable file system hierarchies.
+Since the inception of HeuDiConv in 2014, the BIDS standard [@GAC+16] was established.
+This standard formalizes data file hierarchies and metadata storage in a fashion which, due to its community-driven nature, is both highly optimized and widely understood by analysis tools.
+Since then, DICOM conversion to NIfTI files contained within a BIDS hierarchy has emerged as the most frequent use-case for HeuDiConv.
 
 # Overview of HeuDiConv functionality
 
 ## Exemplar heuristics
 
 ### ReproIn heuristic
 
-ReproIn heuristic was initially developed at Dartmouth Brain Imaging Center (DBIC) to maximally automate data conversion to BIDS for *any* neuroimaging study at the center.
-The principle behind ReproIn is minimization of overall time effort by investing only negligible time at the beginning to organize and name of the MRI programs on the scanner in agreement with ReproIn specification.
-As a result, later transmitted as DICOMs data could *in principle* be fully automatically placed into corresponding study datasets and fully automatically converted.
+The ReproIn heuristic was initially developed at the Dartmouth Brain Imaging Center (DBIC) to automate data conversion into BIDS for any neuroimaging study performed using the center's facilities.
+The core principle behind ReproIn is the reduction of operator interaction requirements by ensuring that reference MRI sequences on the instrumentation are named in such a way that upon usage in any experimental protocol they will encode the information required for fully automatic conversion and repositing of the resulting data.
```

Comment:
```suggestion
The core principle behind ReproIn is the reduction of operator interaction requirements by ensuring that reference MRI sequences on the instrumentation are organized and named in such a way that upon usage in any experimental protocol they will encode the information required for fully automatic conversion and repositing of the resulting data.
```

Reply:
in how far are they additionally “organized” in a way that is reflected in output DICOMs and is different from their naming?

Reply:
organized across studies. HeuDiConv can figure out automagically to which folder/dataset to place converted data, differently across different studies if heuristic tells where to.  We use that in ReproIn heuristic